app/serializers/schedule_serializer.rb

```diff
@@ -0,0 +1,22 @@
+class ScheduleSerializer
+  include JSONAPI::Serializer
+
+  attributes :id, :created_at
+
+  has_many :shows
+
+  attribute :time_slot_map do |schedule|
+    (1..8).map do |slot|
```

Comment:
There's that magic number again - this `MAX_TIME_SLOT` value should probably go in a module or class where everyone can get to it.